documentation/angular-ci.asciidoc

```diff
@@ -99,40 +99,40 @@ In order to easily reuse the pipeline in other angular projects, all variables h
 ----
 environment {
     // Script for build the application. Defined at package.json
-    buildScript = 'build --configuration=docker'
+    `buildScript` = 'build --configuration=docker'
     // Script for lint the application. Defined at package.json
-    lintScript = 'lint'
+    `lintScript` = 'lint'
     // Script for test the application. Defined at package.json
     testScript = 'test:ci'
     // Angular directory
-    angularDir = 'angular'
+    `angularDir` = 'angular'
     // SRC folder. It will be angularDir/srcDir
-    srcDir = 'src'
+    `srcDir = 'src'`
     // Name of the custom tool for chrome stable
     chrome = 'Chrome-stable'
 
-    // sonarQube
-    // Name of the sonarQube tool
-    sonarTool = 'SonarQube'
-    // Name of the sonarQube environment
-    sonarEnv = "SonarQube"
+    // SonarQube
+    // Name of the SonarQube tool
+    `sonarTool` = 'SonarQube'
+    // Name of the SonarQube environment
+    `sonarEnv` = "SonarQube"
 
     // Nexus
     // Artifact groupId
-    groupId = 'com.devonfw.mythaistar'
+    `groupId` = 'com.devonfw.mythaistar'
     // Nexus repository ID
-    repositoryId = 'pl-nexus'
+    `repositoryId` = 'pl-nexus'
     // Nexus internal URL
-    repositoryUrl = 'http://nexus3-core:8081/nexus3/repository/maven-snapshots'
+    `repositoryUrl` = 'http://nexus3-core:8081/nexus3/repository/maven-snapshots'
     // Maven global settings configuration ID
-    globalSettingsId = 'MavenSettings'
+    `globalSettingsId` = 'MavenSettings'
     // Maven tool id
-    mavenInstallation = 'Maven3'
+    `mavenInstallation` = 'Maven3'
```

Comment:
ideally source should not be added to backticks ... so block between

[source]
----
// code here
----
should be ignored